frontend/packages/dev-console/src/components/catalog/providers/useDevfile.tsx

```diff
@@ -11,7 +11,7 @@ const normalizeDevfile = (devfileSamples: DevfileSample[], t: TFunction): Catalo
   const normalizedDevfileSamples = devfileSamples?.map((sample) => {
     const { name: uid, displayName, description, tags, git, icon } = sample;
     const gitRepoUrl = Object.values(git.remotes)[0];
-    const iconUrl = icon ? `data:image/png;base64,${icon}` : '';
+    const iconUrl = icon || '';
```

Comment:
Do we know if the icons will always be URLs instead of base64 encoded images?

Can we use `icon` directly and remove the `iconUrl` var now?

Reply:
Yes. The icons from registry will always be URLs. I have removed `iconUrl` and use `icon` directly